src/app/pages/FeedPage/components/TwitsUnit.tsx

```diff
@@ -0,0 +1,140 @@
+import 'index.css'
+
+import SvgLike from 'app/components/SVG/SvgLike'
+import SvgComment from 'app/components/SVG/SvgComment'
+import SvgRetwit from 'app/components/SVG/SvgRetwit'
+import { Twit } from 'types/Twit'
+import Image from 'app/components/Image'
+
+export type TwitUnitProps = {
+  data: Twit
+}
+
+const defaultAvatarUrl =
+  'data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBw8PCw0QEQ8PChINDQ0IDQ0NCA8ICggOGBEWFhURExUYHSggGBo0GxMVITEhJSkrLjouFx8zODMtNygtLisBCgoKDQ0NDg0NDisZFRkrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrK//AABEIAGQAZAMBIgACEQEDEQH/xAAbAAEAAwEBAQEAAAAAAAAAAAAAAwQFAgEGB//EADAQAAIBAgMHAwMDBQAAAAAAAAABAgMRBBIxBSEiMkFhcVFSYkKBghMj0XKRkqHB/8QAFQEBAQAAAAAAAAAAAAAAAAAAAAH/xAAUEQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIRAxEAPwD9xAOKlRRi23ZLVgdNlHEbShHdH9x9uX+5QxuNlUulwx9Pd5KgFqptCpLrk7JZSvKbercvLzHIKgn9iWniakdJyXa+aJEANGjtWS5kpd1ws0qGIjNXi0/VdYnzh1Cbi003FrRoD6cFHA45T4ZcMv8AUy8RQAAeNmFtDF/qSsuWL3fLuX9q18tPKtZ8P49TFAAAqABbpYCb1tDs+YCoC9LZz6ST8oq1qEocyt6NcsgIwAB6nZp71biTRu7PxX6kN/NHhl37mCTYOvkqKXTll4A+jB4mCKwdp1M1aXx4EVTqpK8pP1bkclQAAGls/D2Sm9ZcvxRcPIqyXZZT0ihzOCkmnvT1R0AMbEUsk2tbaP3IiL21I74P1TiUSoAADe2dPNRjf6f239gZOHrOMbd7giq7WoJcXDLVmvk7ERUAABtYapmpxfbK/JIY+FxLpt9U9UalLEQlo14fDIipAeOSXVLyypiccldR4n7vpiBBtGpeol7Vl+5UDYKgAAPVG4NTZuHzUr+smAI9sUbSU/VZX5M4+jxNFTg4vro/az56pFxk09zTs0ByATYbDOb9qWsgISWOHm9Ivy1lNSjh4w0X5PmJSKyHg6ntb/KJFODjqnHyjcDXh9mBgg0sRgU7uPC/b9MjOlFptO6tqmVHh7GN2kt93lSPDQ2ThryzvRcvdgauHp5YRj6KwJARQo7Qwf6izR5krf1ovAD5uhQcp5d8bc3xNiEUkktyWiLEqSbvo3q0tSKUWv5A5AAAAACrjsPmjdc0V/kvaWiSFL13dgMbBYR1JdVFc0v+I3oQUUktySskIxSVkrW6I6AAAAAAAAA4dNePBFNWAA56kkKaYAEqikegAAAAAAH/2Q=='
+
+export default function TwitsUnit({
+  data: {
+    id,
+    author,
+    content,
+    likes = 0,
+    comments = 0,
+    retwits = 0,
+    createdAt,
+    updatedAt,
+  },
+}: TwitUnitProps) {
+  const avatar = author.avatar ?? defaultAvatarUrl
+
+  function elapsedDate(time: any) {
+    const date = new Date((time || '').replace(/-/g, '/').replace(/[TZ]/g, ' '))
+    const sec_diff = (new Date().getTime() - date.getTime()) / 1000
+    const day_diff = Math.floor(sec_diff / 86400)
+
+    if (isNaN(day_diff) || day_diff < 0 || day_diff >= 31) return
```

Comment:
`If (day_diff >= 31) return` but later at [these lines](https://github.com/KirillKrm/twitter-clone/pull/2/files#diff-615e7a5c0bf4e11c06e4349e9e21e2aabd1a82663d99e555a2bded91112d2458R48-R51) `day_diff` is basically bigger or equal that 31. So this function return `undefined` if date was earlier that 31 days ago. 
Or am i wrong?

Reply:
Yes, you are right. This condition should not be here.